src/sentry/incidents/models/__init__.py

```diff
@@ -0,0 +1,10 @@
+__all__ = (
+    "AlertRule",
+    "AlertRuleActivity",
+    "AlertRuleTrigger",
+    "AlertRuleTriggerAction",
+    "Incident",
+)
+
+from .alert_rule import AlertRule, AlertRuleActivity, AlertRuleTrigger, AlertRuleTriggerAction
+from .incident import Incident
```

Comment:
Why these import changes? Seems a bit unnecessary, especially since these are going away

Reply:
just to keep the way deletion models are imported consistent